rl_loop.py

```diff
@@ -158,13 +157,15 @@ def backfill():
             continue
 
 
-parser = argparse.ArgumentParser()
-
-argh.add_commands(parser, [train, selfplay, backfill,
-                           bootstrap, fsdb.game_counts, validate,
-                           validate_hourly])
-
 if __name__ == '__main__':
     cloud_logging.configure()
     remaining_argv = flags.FLAGS(sys.argv, known_only=True)
-    argh.dispatch(parser, argv=remaining_argv[1:])
+    fire.Fire({
+        'train': train,
+        'selfplay': selfplay,
+        'backfill': backfill,
+        'bootstrap': bootstrap,
+        'fsdb.game_counts': fsdb.game_counts,
```

Comment:
I think this one would be called as `game_counts`, not `fsdb.game_counts`.

Reply:
Done.